src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -283,6 +283,8 @@ public Value get(String what, Value arg)
         put("sneaking", (e, a) -> e.isSneaking()?Value.TRUE:Value.FALSE);
         put("sprinting", (e, a) -> e.isSprinting()?Value.TRUE:Value.FALSE);
         put("swimming", (e, a) -> e.isSwimming()?Value.TRUE:Value.FALSE);
+        put("hunger", (e, a) -> new NumericValue(((PlayerEntity) e).getHungerManager().getFoodLevel()));
```

Comment:
you have to make sure it won't crash if e is not a player - returning null, not npe

Reply:
ah ok, I hadn't tested that